### PR TITLE
Fixes/post delivery tasks

### PR DESCRIPTION
**Pull request by ShrawanBK**
[] Remove the last two charts of regions page (bi-axial and scatterplot)
[] Add two charts to the region page ( Wealth index, and Paved Road Network)
[] Rename DFID to BEK
[] Update action icons in charts
[] Update label font-size in charts
[] Rename 'programs' to 'programmes'
[] Rename Hide all data with 'Hide nation wide data
[] Use Programme acronyms in charts and use the full name of admin names (province/districts and municipalities)

### CODE REVIEW

File: src/components/ChartModal/BarChartConfig/index.tsx [hunk 1]
```diff
@@ -173,6 +174,7 @@ function BarChartConfig<T>(props: Props<T>) {
         maxRow = 4,
         options,
         keySelector: primaryKeySelector,
+        acronymSelector: primaryAcronymSelector,
```
Comment (AdityaKhatri) [inline src/components/ChartModal/BarChartConfig/index.tsx]:
```suggestion
        acronymSelector,
```

File: src/components/ChartModal/BarChartConfig/index.tsx [hunk 2]
```diff
@@ -302,7 +304,7 @@ function BarChartConfig<T>(props: Props<T>) {
                 title,
 
                 keySelector: primaryKeySelector,
-
+                acronymSelector: primaryAcronymSelector,
```
Comment (AdityaKhatri) [inline src/components/ChartModal/BarChartConfig/index.tsx]:
```suggestion
                acronymSelector,
```

File: src/components/ChartModal/BarChartConfig/index.tsx [hunk 3]
```diff
@@ -318,7 +320,7 @@ function BarChartConfig<T>(props: Props<T>) {
         },
         [
             editableChartData, onSave, bars, title, options, barType, limitValue,
-            order, autoOrderField, primaryKeySelector,
+            order, autoOrderField, primaryKeySelector, primaryAcronymSelector,
```
Comment (AdityaKhatri) [inline src/components/ChartModal/BarChartConfig/index.tsx]:
```suggestion
            order, autoOrderField, primaryKeySelector, acronymSelector,
```

File: src/views/Dashboard/RegionDetails/Charts/index.tsx [hunk 4]
```diff
@@ -140,67 +140,69 @@ const defaultChartSettings: ChartSettings<ExtendedFiveW>[] = [
...
-                color: tableauColors[5],
-                valueSelector: item => item.indicators[25] || null,
+                key: 'wealthIndex',
+                title: 'Wealth Index',
+                color: tableauColors[3],
+                valueSelector: item => item.indicators[504] || null,
             },
         ],
-        dependencies: [25],
+        dependencies: [504],
```
Comment (AdityaKhatri) [inline src/views/Dashboard/RegionDetails/Charts/index.tsx]: Lets define the ids as enums and use it here.

File: src/views/Dashboard/RegionDetails/Charts/index.tsx [hunk 5]
```diff
@@ -140,67 +140,69 @@ const defaultChartSettings: ChartSettings<ExtendedFiveW>[] = [
...
         type: 'bar-chart',
-        title: 'Top 10 by poverty incidence',
+        title: 'Top 10 Paved Road Network',
         keySelector: item => item.name,
 
         limit: {
             count: 10,
             method: 'max',
-            valueSelector: item => item.indicators[132] || null,
+            valueSelector: item => item.indicators[156] || null,
```
Comment (AdityaKhatri) [inline src/views/Dashboard/RegionDetails/Charts/index.tsx]: Lets define the ids as enums and use it here.

File: src/views/Dashboard/RegionDetails/Charts/index.tsx [hunk 6]
```diff
@@ -140,67 +140,69 @@ const defaultChartSettings: ChartSettings<ExtendedFiveW>[] = [
...
 
         bars: [
             {
-                title: 'Poverty Incidence',
-                color: tableauColors[6],
-                valueSelector: item => item.indicators[132] || null,
+                key: 'pavedRoadNetworn',
+                title: 'Paved Road Network',
+                color: tableauColors[0],
+                valueSelector: item => item.indicators[156] || null,
```
Comment (AdityaKhatri) [inline src/views/Dashboard/RegionDetails/Charts/index.tsx]: Lets define the ids as enums and use it here.

File: src/views/Dashboard/RegionDetails/Charts/index.tsx [hunk 7]
```diff
@@ -140,67 +140,69 @@ const defaultChartSettings: ChartSettings<ExtendedFiveW>[] = [
...
-    {
-        id: '4',
-        type: 'histogram',
-        key: 'componentCount',
-        title: 'Component Count',
-        color: tableauColors[0],
-        binCount: 10,
-        valueSelector: item => item.componentCount,
-        // dependencies: [118],
+        dependencies: [156],
```
Comment (AdityaKhatri) [inline src/views/Dashboard/RegionDetails/Charts/index.tsx]: Lets define the ids as enums and use it here.

File: src/views/Dashboard/RegionDetails/Charts/index.tsx [hunk 8]
```diff
@@ -140,67 +140,69 @@ const defaultChartSettings: ChartSettings<ExtendedFiveW>[] = [
...
         type: 'bar-chart',
-        title: 'Top 10 by population',
+        title: 'Top 10 by Mean International Wealth Index',
         keySelector: item => item.name,
 
         limit: {
             count: 10,
             method: 'max',
-            valueSelector: item => item.indicators[25] || null,
+            valueSelector: item => item.indicators[504] || null,
```
Comment (AdityaKhatri) [inline src/views/Dashboard/RegionDetails/Charts/index.tsx]: Lets define the ids as enums and use it here.

File: src/components/PolyChart/BarChart/index.tsx [hunk 9]
```diff
@@ -74,14 +68,37 @@ const renderCustomizedLabel = (verticalLayout: boolean) => (props: CustomizedLab
...
+                fontSize="0.8rem"
             >
                 {valueTickFormatter(value)}
             </text>
         </g>
     );
 };
 
+const CustomTooltip = ({ active, payload }: TooltipProps) => {
+    if (!payload || payload.length <= 0) { return null; }
```
Comment (AdityaKhatri) [inline src/components/PolyChart/BarChart/index.tsx]:
```suggestion
    if (!payload || payload.length <= 0) {
        return null;
    }
```

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: src/components/PolyChart/BarChart/index.tsx [hunk 10]
```diff
@@ -36,10 +38,12 @@ const orientations: {
...
     const words = value.trim().split(/\s+/);
-    if (words.length <= 1) {
-        return value.slice(0, 3).toUpperCase();
-    }
-    return words.map(item => item[0]).join('').toUpperCase();
+    // NOTE: words "Province", "District", "Municipality" removed from tickformatter for better UI
+    const administrationIndex = words.findIndex(
+        i => caseInsensitiveSubmatch(i, 'Province') || caseInsensitiveSubmatch(i, 'District') || caseInsensitiveSubmatch(i, 'Municipality'),
+    );
+    if (administrationIndex <= 0) return words.join(' ');
```
Comment (tnagorra) [inline src/components/PolyChart/BarChart/index.tsx]: Always add block for if/else blocks

File: src/components/PolyChart/BarChart/index.tsx [hunk 11]
```diff
@@ -74,14 +68,37 @@ const renderCustomizedLabel = (verticalLayout: boolean) => (props: CustomizedLab
...
+    if (!payload || payload.length <= 0) { return null; }
+    const {
+        name: legendName,
+        payload: {
+            name: label,
+        },
+        value,
+        color,
+    } = payload[0];
+    if (active) {
```
Comment (tnagorra) [inline src/components/PolyChart/BarChart/index.tsx]: It's easier to shortcircuit

File: src/components/PolyChart/BiAxialChart/index.tsx [hunk 12]
```diff
@@ -25,10 +26,12 @@ import styles from './styles.css';
...
     const words = value.trim().split(/\s+/);
-    if (words.length <= 1) {
-        return value.slice(0, 3).toUpperCase();
-    }
-    return words.map(item => item[0]).join('').toUpperCase();
+    // NOTE: words "Province", "District", "Municipality" removed from tickformatter for better UI
+    const administrationIndex = words.findIndex(
+        i => caseInsensitiveSubmatch(i, 'Province') || caseInsensitiveSubmatch(i, 'District') || caseInsensitiveSubmatch(i, 'Municipality'),
+    );
+    if (administrationIndex <= 0) return words.join(' ');
```
Comment (tnagorra) [inline src/components/PolyChart/BiAxialChart/index.tsx]: Always add if/else block (body with curly braces)

File: src/components/PolyChart/BiAxialChart/index.tsx [hunk 13]
```diff
@@ -25,10 +26,12 @@ import styles from './styles.css';
 
 const categoryTickFormatter = (value: string) => {
     const words = value.trim().split(/\s+/);
-    if (words.length <= 1) {
-        return value.slice(0, 3).toUpperCase();
-    }
-    return words.map(item => item[0]).join('').toUpperCase();
+    // NOTE: words "Province", "District", "Municipality" removed from tickformatter for better UI
+    const administrationIndex = words.findIndex(
+        i => caseInsensitiveSubmatch(i, 'Province') || caseInsensitiveSubmatch(i, 'District') || caseInsensitiveSubmatch(i, 'Municipality'),
```
Comment (tnagorra) [inline src/components/PolyChart/BiAxialChart/index.tsx]: Why not use regex?

File: src/components/PolyChart/BiAxialChart/index.tsx [hunk 14]
```diff
@@ -40,6 +43,37 @@ const valueTickFormatter: TickFormatterFunction = (value) => {
...
+        name: firstName,
+        value: firstValue,
+        color: firstColor,
+    } = payload[0];
+    const {
+        name: secondName,
+        value: secondValue,
+        color: secondColor,
+    } = payload[1];
+    if (active) {
```
Comment (tnagorra) [inline src/components/PolyChart/BiAxialChart/index.tsx]: It's better to shortcircuit

**Review by tnagorra — CHANGES_REQUESTED** [review]

**Review by AdityaKhatri — APPROVED** [review]

**Pull request merged**